mesonbuild/utils/universal.py

```diff
@@ -1086,6 +1088,29 @@ def default_sysconfdir() -> str:
     return 'etc'
 
 
+def determine_worker_count(varnames: T.Optional[T.List[str]] = None) -> int:
+    num_workers = 0
+    varnames = varnames or []
+    varnames.append('MESON_NUM_PROCESSES')
+    for varname in varnames:
+        if varname in os.environ:
```

Comment:
So the idea here is to use the last varname that exists?

Can we document the order used here?